connectors/filesystem.mdx

```diff
@@ -44,21 +57,42 @@ Parquet has a few supported options. Please reach out if you'd like to expand th
 | `parquet_row_batch_size` | The maximum number of rows to write per record batch | `10000` | `100`|
 | `parquet_row_group_size` | The maximum number of rows to write per row group | `1000000` | `100000` |
 
+### Partitioning Options
+Arroyo now supports partitioning of outputs. There are two types of partitioning: event time-based and field-based.
```

Comment:
I think we can remove "now" since readers won't really care that it wasn't supported in the past